spec/models/course/reference_time_spec.rb

```diff
@@ -86,5 +86,40 @@
         end
       end
     end
+
+    describe '#reset_closing_reminders on assessment' do
```

Comment:
By the way i think you can add a check that the closing reminder token has been updated too